engines/e_afalg.c

```diff
@@ -224,7 +224,7 @@ static int afalg_fin_cipher_aio(afalg_aio *aio, int sfd, unsigned char *buf,
     struct iocb *cb;
     struct timespec timeout;
     struct io_event events[MAX_INFLIGHTS];
-    u_int64_t eval = 0;
+    uint64_t eval = 0;
```

Comment:
I was just trying to  clear out the simple issue.  You want to take this on and fix the real issue?

Reply:
Basically objection is same as in moving e_os.h. It's not really an issue because code is not portable and target platform does have the type(*), while addressing it and disregarding real problem so to say "legitimates" the practice of disregarding. As for undertaking the real issue, I'd rather not, I'm not that intimately familiar with interface in question and more importantly control flow...

(*)  The referred #6097 speaks of QNX, but this code is not supposed to be exposed to QNX compiler. If it is, then it ought to be a bug, then should *that* be fixed? But of course it's not actually exposed to QNX, because there are even Linux-specific structures. So obviously reporter simply searched for for u_int... But in such case why just this file? Code that is *not* exposed to QNX compiler?